setup.cfg

```diff
@@ -1,3 +1,59 @@
+[tool:pytest]
+testpaths =
+    mint/
+addopts =
+    -ra
+    -v
```

Comment:
@pletzer This configures `pytest` to run the `develop` installed version of `mint`